colossalai/trainer/hooks/_lr_scheduler_hook.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         self.store_lr_in_state = store_lr_in_state
 
     def after_hook_is_attached(self, trainer):
+        self._check_metric_states_initialization(trainer)
```

Comment:
Is this line of code related to the profiler?

Reply:
A small bug should be fixed when using learning rate scheduler alone.